src/benchmarking/openai_bert_score.py

```diff
@@ -0,0 +1,77 @@
+from dataclasses import dataclass
+
+import numpy as np
+import tiktoken
+
+from langchain_openai import OpenAIEmbeddings
+from sklearn.metrics.pairwise import cosine_similarity
+
+
+@dataclass
+class BERTScoreResult:
+    precision: float
+    recall: float
+    f1: float
+
+
+class BERTScore:
```

Comment:
Technically, BERT is a language model, so BERTScore is a BERTScore iff it is measured by BERT. More precise term would be "semantic similarity"